components/contribution-flow/ContributionBlocker.js

```diff
@@ -49,6 +50,16 @@ const msg = defineMessages({
 /**
  * From received params, see if there's anything preventing the contribution
  */
+const Redirect = ({ to }) => {
+  const router = useRouter();
+
+  useEffect(() => {
+    router.push(to);
+  }, [to]);
+
+  return null;
```

Comment:
It would be nice to show a loading spinner while the user is being redirected so that the page doesn't look empty

```suggestion
  return <Loading />;
```